### PR TITLE
Add antialiasing and hinting

### PR DESCRIPTION
**Pull request by strausr**
Add antialiasing and hinting

### CODE REVIEW

File: lib-es5/uploader.js [hunk 1]
```diff
@@ -241,7 +241,7 @@ exports.rename = function rename(from_public_id, to_public_id, callback) {
   });
 };
 
-var TEXT_PARAMS = ["public_id", "font_family", "font_size", "font_color", "text_align", "font_weight", "font_style", "background", "opacity", "text_decoration", "font_hinting", "font_antialiasing"];
+var TEXT_PARAMS = ["public_id", "font_family", "font_size", "font_color", "text_align", "font_weight", "font_style", "background", "opacity", "text_decoration"];
```
Comment (asisayag2) [inline lib-es5/uploader.js]: @strausr , the params were removed

File: lib/utils/index.js [hunk 2]
```diff
@@ -159,25 +157,25 @@ function textStyle(layer) {
...
+  Object.keys(layer).forEach((attr) => {
+    if (attr === "letter_spacing" || attr === "line_spacing") {
+      keywords.push(`${attr}_${layer[attr]}`);
+    }
+    if (attr === "font_antialiasing" || attr === "font_hinting") {
+      keywords.push(`${attr.split("_").pop()}_${layer[attr]}`);
     }
-    if (!font_size) {
-      throw "Must supply font_size for text in overlay/underlay";
+    if (attr || !isEmpty(keywords)) {
```
Comment (asisayag2) [inline lib/utils/index.js]: @strausr , I don't understand how this works

File: test/utils_spec.js [hunk 3]
```diff
@@ -1107,6 +1107,21 @@ describe("utils", function () {
             },
             `text:Arial_18_bold_italic_letter_spacing_4_line_spacing_2:${text_encoded}`,
           ],
+          [
+            "text with text font_antialiasing and font_hinting",
```
Comment (asisayag2) [inline test/utils_spec.js]: @strausr , as discussed, let's add a separate test for the antialiasing and for hinting

File: lib-es5/utils/index.js [hunk 4]
```diff
@@ -1493,4 +1501,4 @@ exports.build_streaming_profiles_param = build_streaming_profiles_param;
 exports.hashToParameters = hashToParameters;
 exports.present = present;
 exports.only = only;
-exports.jsonArrayParam = jsonArrayParam;
\ No newline at end of file
+exports.jsonArrayParam = jsonArrayParam;
```
Comment (asisayag2) [inline lib-es5/utils/index.js]: @strausr , why was that added?

File: lib/uploader.js [hunk 5]
```diff
@@ -189,7 +189,7 @@ exports.rename = function rename(from_public_id, to_public_id, callback, options
   });
 };
 
-const TEXT_PARAMS = ["public_id", "font_family", "font_size", "font_color", "text_align", "font_weight", "font_style", "background", "opacity", "text_decoration"];
+const TEXT_PARAMS = ["public_id", "font_family", "font_size", "font_color", "text_align", "font_weight", "font_style", "background", "opacity", "text_decoration", "font_hinting", "font_antialiasing"];
```
[the remark this answers is not in the document]
Comment (asisayag2) [reply lib/uploader.js]: @strausr , was that added to lib-es5?

File: lib/utils/index.js (hunk 2 above)
Comment (strausr) [reply lib/utils/index.js]: changed logic in new commit

File: lib-es5/utils/index.js (hunk 4 above)
Comment (strausr) [reply lib-es5/utils/index.js]: Added in the es5 compile automatically
